package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "teleport-ui",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "build-ui": "yarn build-ui-oss && yarn build-ui-e",
+    "build-ui-oss": "yarn workspace @gravitational/teleport build --output-path=../../../webassets/teleport/app",
+    "build-ui-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../webassets/e/teleport/app",
+    "build-teleport": "yarn workspace @gravitational/namespaces build-teleport",
+    "start-teleport": "yarn workspace @gravitational/namespaces start-teleport",
+    "build-term": "yarn workspace @gravitational/namespaces build-term",
+    "start-term": "yarn workspace @gravitational/namespaces start-term",
+    "package-term": "yarn workspace @gravitational/namespaces package-term",
+    "build-native-deps-for-term": "yarn workspace @gravitational/namespaces build-native-deps-for-term",
+    "build-and-package-term-ci": "yarn workspace @gravitational/namespaces build-and-package-term-ci",
+    "test": "yarn workspace @gravitational/namespaces test",
+    "type-check": "yarn workspace @gravitational/namespaces type-check"
```

Comment:
i notice there's no lint or storybook scripts here, which are present on master as well as in the webapps repo on the v11 branch, are they missing or not needed?

Reply:
Sorry those are coming in future pr's. I'm applying the changes in the same order as was done to `master` and `branch/v12`. Some of the future ones though will combine a few PR's.